deploy/src/main/sandbox-k8s/10-deploy-cert-manager.sh

```diff
@@ -12,43 +12,20 @@
 # SPDX-License-Identifier: EPL-2.0
 #*******************************************************************************
 
-# This script takes care to provide a public certificate from Let's Encrypt.
-# It creates the namespace "hono", deploys the Cert-Manager and requests a certificate.
-# The script expects an email address to be used by Let's Encrypt, as the first argument.
-# The second argument must be "production" to create a valid certificate. If it is not set,
-# the issuer will be configured to use the Let's Encrypt Staging API. This is to avoid
-# hitting a limit on Let's Encrypt's productive API.
+# This script prepares the certificate management. It deploys the Cert-Manager and creates the namespace "hono".
 
-SCRIPTPATH="$(cd "$(dirname "$0")" && pwd -P)"
-source $SCRIPTPATH/common.sh
+set -ue
 
-NS_CERT_MANAGER=cert-manager
-EMAIL=$1
-LE_API=${2:-"staging"}
 
-
-retry() {
-  "$@"
-  while [ $? -ne 0 ]; do
-    sleep 2s
-    echo "retrying" "$@"
-    "$@"
-  done
-}
-
-kubectl create namespace hono $KUBECONFIG
-
-# add Helm repo
-helm repo add jetstack https://charts.jetstack.io $KUBECONFIG
-helm repo update $KUBECONFIG
+echo ""
+echo "Adding the Helm repository for cert-manager..."
+helm repo add jetstack https://charts.jetstack.io
+helm repo update
 
 echo ""
 echo "Deploying cert-manager..."
-helm install cert-manager jetstack/cert-manager --namespace $NS_CERT_MANAGER --create-namespace --set installCRDs=true $KUBECONFIG $HELM_WAIT #  --version v1.4.0
+helm install cert-manager jetstack/cert-manager --namespace cert-manager --create-namespace --set installCRDs=true --wait --timeout 5m0s
 
-# checking when cert-manager is ready is tricky: https://cert-manager.io/docs/installation/kubernetes/#verifying-the-installation
-# we just retry the following command
 echo ""
-echo "Adding issuer Let's Encrypt to cert-manager with API: $LE_API"
-LE_CONFIG=$(helm template ./letsencrypt-chart -f ./letsencrypt-chart/values-$LE_API.yaml --set spec.acme.email=$EMAIL)
-retry echo "$LE_CONFIG" | kubectl apply -f - $KUBECONFIG || true # do not abort although '-e' is set
+echo "Creating namespace 'hono'..."
+kubectl create namespace hono
```

Comment:
why do we create the *hono* namespace here? This doesn't seem to be related to the cert-manager, is it?

Reply:
Right, it seems a little misplaced. Therefore it is explicitly mentioned in the comments of the script and in the readme. The point is that the following script needs the namespace to create the secret for the certificate in it. Having an additional script only for creating the namespace seemed a little bit like overkill to me. But maybe that would be cleaner...

Reply:
why don't we simply mention it in the readme that you need to create the namespace using `kubectl create namespace hono`?

Reply:
Done.